Fix Bugs in Sorting Problem, issue #21

gauravghodinde opened the pull request:
removed unnecessary variables and while loop in main function in issue #21 

Screenshot of output
![image](https://github.com/DSC-IIITL/GitGrind/assets/65962770/2d8adef0-960f-484b-92b7-bcf893d2f7c4)

akhilender-bongirwar reviewed (changes requested):
@gauravghodinde, remove the .exe file. solve it asap

gauravghodinde commented:
@akhilender-bongirwar which .exe file are you referring to I removed all the exe files in the last commit.
the fibonacci.exe file was present in the previous commit but in the lastest(final) commit i removed it

Illuminati9 reviewed (approved):
LGTM
